packages/reactotron-core-ui/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "reactotron-core-ui",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/reactotron-core-ui.js",
+  "typings": "dist/types/reactotron-core-ui.d.ts",
+  "scripts": {
+    "compile": "parcel build src/reactotron-core-ui.ts",
+    "compile:ts": "tsc",
+    "clean": "rimraf dist build",
+    "lint": "tslint -p .",
+    "build": "yarn clean && yarn compile && yarn compile:ts"
```

Comment:
Parcel doesn't put out defintions... so we do both tsc and parcel. Yay for double compiling!